src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/GraphQLMiddleware.cs

```diff
@@ -165,13 +165,10 @@ private async Task ExecuteAsync(HttpContext context, ISchemaFactory schemaServic
                 _.Inputs = request.Variables.ToInputs();
                 _.UserContext = _settings.BuildUserContext?.Invoke(context);
                 _.ExposeExceptions = _settings.ExposeExceptions;
+                _.ValidationRules = (_settings.ValidationRules ?? Enumerable.Empty<IValidationRule>()).Concat(DocumentValidator.CoreRules());
```

Comment:
no need to do ?? here... _settings.ValidationRules is a list and will never be null.